lib/base/events.js

```diff
@@ -96,10 +96,12 @@ class Events extends EventEmitter {
    */
   triggerThen(nameOrNames) {
     const names = eventNames(nameOrNames);
-    const listeners = _.flatMap(names, name => this.listeners(name));
+    const listeners = _.flatMap(names, (name) => this.listeners(name));
     const args = Array.from(arguments);
 
-    return Promise.mapSeries(listeners, listener => listener.apply(this, args.slice(1)));
+    return Promise.mapSeries(listeners, (listener) =>
+      listener.apply(this, args.slice(1))
```

Comment:
This is weird. Splitting a single line fat arrow function into multiple lines makes its intention a bit ambiguous: e.g. was it supposed to be returning something or was it supposed to return `undefined`? Did the author made a mistake?